draft-tigress-requirements.md

```diff
@@ -99,25 +99,25 @@ informative:
 
 --- middle
 
 # Introduction
 
-Today, there is no widely accepted way of transferring digital credentials securely between two devices belonging to the same platform or two different platforms. This document describes the problem space and the requirements for the solution the working group creates.
+Today, there is no widely accepted way of transferring digital credentials securely between two digital wallets independent of hardware and software manufacturer. This document describes the problem space and the requirements for the solution the working group creates.
 
-Tigress allows for a sender and receiver device to communicate in order to facilitate a secure credential transfer. Tigress also specifies certain privacy requirements in order to maintain a high level of user privacy.
+Tigress allows for a sender and receiver to communicate in order to facilitate a secure credential transfer between two digital wallets. Tigress also specifies certain privacy requirements in order to maintain a high level of user privacy.
 
 # General Setting
 
-When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two devices, potentially through an intermediary server, there are a couple more actors involved.
+When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two digital wallets, potentially through an intermediary server, there are a couple more actors involved.
 
-The companies that are providing the digital credential are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. Each device has a preexisting trust relationship between itself and the Provisioning Partner.
+The companies that are providing the digital credential for consumption by a digital wallet are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. Each digital wallet has a preexisting trust relationship between itself and the Provisioning Partner.
 
-The interface between the devices and the Provisioning Partner can be proprietary or part of published specifications such as the {{CCC-Digital-Key-30}}. The sender device obtains provisioning information from the provisioning partner, then shares it to the recipient device via Tigress. The recipient then takes that data and sends it to the Provisioning Partner to redeem a credential.
+The interface between the devices and the Provisioning Partner can be proprietary or part of a public standard such as the CCC. The sender obtains provisioning information from the provisioning partner, then shares it to the recipient via Tigress. The recipient then takes that data and sends it to the Provisioning Partner to redeem a credential for consumption in a digital wallet.
```

Comment:
this undid the CCC piece that Dmitry just edited. Have to add that back and make it "published specification".